From 41691f8bc9bf034ad54e6d3d3f668fbfaeeb1a28 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 18 Jun 2024 16:05:36 +0200
Subject: [PATCH] module-vban: fix some implicit float conversions

---
 src/modules/module-vban/stream.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/modules/module-vban/stream.c b/src/modules/module-vban/stream.c
index 64fc644b0..ed32f3bd5 100644
--- a/src/modules/module-vban/stream.c
+++ b/src/modules/module-vban/stream.c
@@ -361,12 +361,12 @@ struct vban_stream *vban_stream_new(struct pw_core *core,
 	min_samples = (uint32_t)(min_ptime * impl->rate / 1000);
 	max_samples = SPA_MIN(256u, (uint32_t)(max_ptime * impl->rate / 1000));
 
-	float ptime = 0;
+	float ptime = 0.f;
 	if ((str = pw_properties_get(props, "vban.ptime")) != NULL)
 		if (!spa_atof(str, &ptime))
-			ptime = 0.0;
+			ptime = 0.0f;
 
-	if (ptime) {
+	if (ptime != 0.f) {
 		impl->psamples = (uint32_t)(ptime * impl->rate / 1000);
 	} else {
 		impl->psamples = impl->mtu / impl->stride;
-- 
GitLab

