From 51bf143a7765e0c818f51132258a458c549ec8fb Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 18 Jun 2024 16:01:41 +0200
Subject: [PATCH] module-rtp: fix fmodf usage

---
 src/modules/module-rtp/stream.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/module-rtp/stream.c b/src/modules/module-rtp/stream.c
index 24722c0e8..30664b830 100644
--- a/src/modules/module-rtp/stream.c
+++ b/src/modules/module-rtp/stream.c
@@ -507,7 +507,7 @@ struct rtp_stream *rtp_stream_new(struct pw_core *core,
 	}
 
 	/* We're not expecting odd ptimes, so this modulo should be 0 */
-	if (fmodf(impl->target_buffer, ptime != 0)) {
+	if (fmodf(impl->target_buffer, ptime) != 0) {
 		pw_log_warn("sess.latency.msec should be an integer multiple of rtp.ptime");
 		impl->target_buffer = (uint32_t)((impl->target_buffer / ptime) * impl->psamples);
 	}
-- 
GitLab

