From 76417fd2a6c10a319ab8eb4cc3edde0034f9300a Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Thu, 3 Feb 2022 16:21:41 +0100
Subject: [PATCH] alsa: use MONOTONIC clock for tstamp

---
 spa/plugins/alsa/alsa-pcm.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/spa/plugins/alsa/alsa-pcm.c b/spa/plugins/alsa/alsa-pcm.c
index 99be668c5..05af22512 100644
--- a/spa/plugins/alsa/alsa-pcm.c
+++ b/spa/plugins/alsa/alsa-pcm.c
@@ -1516,8 +1516,10 @@ static int set_swparams(struct state *state)
 	/* get the current params */
 	CHECK(snd_pcm_sw_params_current(hndl, params), "sw_params_current");
 
-	CHECK(snd_pcm_sw_params_set_tstamp_mode(hndl, params, SND_PCM_TSTAMP_ENABLE), "sw_params_set_tstamp_mode");
-
+	CHECK(snd_pcm_sw_params_set_tstamp_mode(hndl, params, SND_PCM_TSTAMP_ENABLE),
+			"sw_params_set_tstamp_mode");
+	CHECK(snd_pcm_sw_params_set_tstamp_type(hndl, params, SND_PCM_TSTAMP_TYPE_MONOTONIC),
+			"sw_params_set_tstamp_type");
 #if 0
 	snd_pcm_uframes_t boundary;
 	CHECK(snd_pcm_sw_params_get_boundary(params, &boundary), "get_boundary");
-- 
GitLab

