From a76d262db6344bf3745782279b548e747ef9a63d Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Tue, 18 Jun 2024 15:47:02 +0200
Subject: [PATCH] modules: fix compilation

---
 src/modules/module-rtp/stream.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/module-rtp/stream.c b/src/modules/module-rtp/stream.c
index a2cfc2012..24722c0e8 100644
--- a/src/modules/module-rtp/stream.c
+++ b/src/modules/module-rtp/stream.c
@@ -465,7 +465,7 @@ struct rtp_stream *rtp_stream_new(struct pw_core *core,
 		if (!framecount) {
 			impl->psamples = msec_to_samples(impl, ptime);
 			pw_properties_setf(props, "rtp.framecount", "%u", impl->psamples);
-		} else if (!ptime) {
+		} else if (ptime == 0.0f) {
 			impl->psamples = framecount;
 			pw_properties_set(props, "rtp.ptime",
 					spa_dtoa(tmp, sizeof(tmp),
-- 
GitLab

