diff -uraN pipewire-1.6.2/src/daemon/systemd/user/filter-chain.service.in omv-pipewire-1.6.2/src/daemon/systemd/user/filter-chain.service.in
--- pipewire-1.6.2/src/daemon/systemd/user/filter-chain.service.in	2026-03-16 12:54:17.000000000 +0100
+++ omv-pipewire-1.6.2/src/daemon/systemd/user/filter-chain.service.in	2026-03-21 20:52:12.850521592 +0100
@@ -1,5 +1,6 @@
 [Unit]
 Description=PipeWire filter chain daemon
+ConditionUser=!@system
 
 After=pipewire.service pipewire-session-manager.service
 BindsTo=pipewire.service
diff -uraN pipewire-1.6.2/src/daemon/systemd/user/pipewire-pulse.service.in omv-pipewire-1.6.2/src/daemon/systemd/user/pipewire-pulse.service.in
--- pipewire-1.6.2/src/daemon/systemd/user/pipewire-pulse.service.in	2026-03-16 12:54:17.000000000 +0100
+++ omv-pipewire-1.6.2/src/daemon/systemd/user/pipewire-pulse.service.in	2026-03-21 20:53:11.065606104 +0100
@@ -15,6 +15,8 @@
 # socket-service relationship, see systemd.socket(5).
 Requires=pipewire-pulse.socket
 ConditionUser=!root
+ConditionUser=!sddm
+ConditionUser=!@system
 Wants=pipewire.service pipewire-session-manager.service
 After=pipewire.service pipewire-session-manager.service
 BindsTo=pipewire.service
diff -uraN pipewire-1.6.2/src/daemon/systemd/user/pipewire.service.in omv-pipewire-1.6.2/src/daemon/systemd/user/pipewire.service.in
--- pipewire-1.6.2/src/daemon/systemd/user/pipewire.service.in	2026-03-16 12:54:17.000000000 +0100
+++ omv-pipewire-1.6.2/src/daemon/systemd/user/pipewire.service.in	2026-03-21 20:53:39.933677915 +0100
@@ -1,5 +1,8 @@
 [Unit]
 Description=PipeWire Multimedia Service
+ConditionUser=!root
+ConditionUser=!sddm
+ConditionUser=!@system
 
 # We require pipewire.socket to be active before starting the daemon, because
 # while it is possible to use the service without the socket, it is not clear
