diff -uraN pipewire-0.3.29/meson.build omv-pipewire-0.3.29/meson.build
--- pipewire-0.3.29/meson.build	2021-06-03 12:01:53.000000000 +0200
+++ omv-pipewire-0.3.29/meson.build	2021-06-03 21:12:16.539833282 +0200
@@ -344,8 +344,8 @@
   cdata.set('HAVE_GSTREAMER_DEVICE_PROVIDER', 1)
 endif
 
-webrtc_dep = dependency('webrtc-audio-processing',
-  version : ['>= 0.2', '< 1.0'],
+webrtc_dep = dependency('webrtc-audio-processing-1',
+  version : ['>= 1.0'],
   required : get_option('echo-cancel-webrtc'))
 
 if webrtc_dep.found()
